app/src/openxr/cpp/OpenXRInputMappings.h

```diff
@@ -215,6 +215,31 @@ namespace crow {
             },
     };
 
+    const OpenXRInputMapping PicoNeo3 {
+            "/interaction_profiles/pico/neo3_controller",
+            IS_6DOF,
+            "vr_controller_pico3_left.obj",
+            "vr_controller_pico3_right.obj",
+            device::PicoNeo3,
+            std::vector<OpenXRInputProfile> { "pico-3", "generic-trigger-squeeze-thumbstick" },
```

Comment:
I don't think this is correct. The correct profile name is pico-neo3. See https://github.com/immersive-web/webxr-input-profiles/blob/main/packages/registry/profiles/pico/pico-neo3.json